tests/roles/development_environment/files/pre_launch.bash

```diff
@@ -17,6 +17,12 @@ function wait_for_status() {
     done
 }
 
+# Benchmark CPU
+# Install sysbench
+ssh -i {{ edpm_privatekey_path }} -o StrictHostKeyChecking=no {{ source_osp_ssh_user }}@{{ standalone_ip | default(edpm_node_ip) }} sudo dnf install -y sysbench
```

Comment:
https://logserver.rdoproject.org/42/642/f78b05eee0fd439da456af41b010f46b064aada5/github-check/adoption-standalone-to-crc-ceph/61fb04c/controller/data-plane-adoption-tests-repo/data-plane-adoption/tests/logs/test_with_ceph_out_2024-10-07T09%3A15%3A54EDT.log

"+ export 'OPENSTACK_COMMAND=ssh -i ~/install_yamls/out/edpm/ansibleee-ssh-key-id_rsa -o StrictHostKeyChecking=no root@192.168.122.100 OS_CLOUD=standalone openstack'", "+ OPENSTACK_COMMAND='ssh -i ~/install_yamls/out/edpm/ansibleee-ssh-key-id_rsa -o StrictHostKeyChecking=no root@192.168.122.100 OS_CLOUD=standalone openstack'", "+ set -e", "+ alias 'openstack=ssh -i ~/install_yamls/out/edpm/ansibleee-ssh-key-id_rsa -o StrictHostKeyChecking=no root@192.168.122.100 OS_CLOUD=standalone openstack'", "/bin/bash: -c: line 25: syntax error near unexpected token `edpm_node_ip'"], "stdout": "", "stdout_lines": []}

Reply:
I guess here we need to use the OPENSTACK_COMMAND instead of the ansible vars

Reply:
@gibizer @bogdando the `edpm_node_ip` is not available in this script. If we need to pass variables we'll need to set the as env vars in https://github.com/openstack-k8s-operators/data-plane-adoption/blob/01b96fc6816fee1e7780ac3892b407edb0a5a595/tests/roles/development_environment/tasks/main.yaml#L1

Reply:
since we don't expect to merge this and it's just for debug, I would simply hardcode the values to use. Otherwise, you could duplicate the `openstack_command` var to the same thing without the `openstack` call at the end